lib/bowling.rb

```diff
@@ -0,0 +1,30 @@
+class BowlingScore
+  attr_reader :current_score
+
+  def initialize
+    @number_of_rolls = 0
+    @current_score = 0
+  end
+
+  def hit(number)
+    @number_of_rolls += 1
+    @current_score += number
+      if @current_score == 10
+        spare
+      end
+  end
+
+  def gutter
+    @current_score += 0
```

Comment:
Nice break down of individual states of the game with separate methods for gutter, spare, etc. 

Missing logic but initial methods look to be on the right track 👍🏼